plugins/app/src/commands.rs

```diff
@@ -12,8 +12,7 @@ pub fn name<R: Runtime>(app: AppHandle<R>) -> String {
 
 #[tauri::command]
 pub fn tauri_version() -> &'static str {
-    // TODO: return actual tauri version with `tauri::VERSION`
-    env!("CARGO_PKG_VERSION")
+    tauri::VERSION
```

Comment:
should we change this to use the newly added `tauri::version()`? https://github.com/tauri-apps/tauri/commit/7c237209207cd2938df660b6fd87d3b7d728bd03

Reply:
or should we change the api in `dev` (still unreleased) to a `const` variable like `next`?

Reply:
It should be a const unless we parse the version string to semver::Version IMO

Reply:
cool, I will update the `dev` branch then